src/s1reader/s1_burst_slc.py

```diff
@@ -781,7 +787,93 @@ def az_fm_rate_mismatch_mitigation(self,
                                     length_radargrid,
                                     width_radargrid,
                                     self.as_isce3_radargrid().ref_epoch)
+        
+        # Run topo on scratch directory
+        dem_raster = isce3.io.Raster(path_dem)
```

Comment:
I am wondering if it is more transparent for the users (and for testing) if we directly assign a DEM raster instead of a path. Also, we have no check at this stage that the DEM really exists. Probably, we should add one

Reply:
@vbrancat I think it's a good idea to add a check at this point. @LiangJYu Thanks for the code suggestion.
On top of Liang's preference, in user perspective, I think it's simpler to provide the path to the DEM because we can put couple of routines (loading DEM as ISCE3 raster, file existence check) into one place.